src/libraries/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs

```diff
@@ -2097,15 +2097,23 @@ private ReadOnlyCollection<TKey> GetKeys()
                     ThrowHelper.ThrowOutOfMemoryException();
                 }
 
-                var keys = new List<TKey>(count);
+                if (count == 0)
+                {
+                    // TODO https://github.com/dotnet/runtime/pull/76097: Replace with ReadOnlyCollection<TKey>.Empty.
```

Comment:
This TODO is pointing to this PR, is that intended?

Reply:
It should have been https://github.com/dotnet/runtime/issues/76028.